### ISSUE TITLE
Adding non-technical 'tips/tricks/etc' type pages?

### ISSUE THREAD

**Issue by cr08** [Cellivar/ford-c-max#2]
Ultimately I'm not sure your intended end goal for this. I had started brainstorming doing my own similarly laid out repo but focusing mostly on more non-technical tips and info about the vehicle more as a quick FAQ of sorts to reference for new/uneducated owners with the big things that always come up for these vehicles. Stuff like the battery health test procedure, best driving practices for managing the battery (Energi of course), notes about some of the common 12v batter drain issues, etc..

I'm more than willing to help put that stuff here if that is an intended goal. At this point I'm still mostly in the brainstorming process on what I've been working on (Figuring out the layout/formatting is my biggest headache at the moment).

**Comment by Cellivar** [comment]
Yes! Guides and how-to docs are totally within scope of the "central place for notes about C-Maxes" that I intended this to be. You [can see an example here](https://github.com/Cellivar/ford-c-max/tree/main/maintenance/procedures) for the BMS reset I recently needed to do. I also [added a fuse guide](https://github.com/Cellivar/ford-c-max/blob/main/systems/electrical/fuses.md).

If you're worried about formatting you're welcome to open a PR with mostly rough text. Markdown takes some getting used to, and time to write the raw documents is what I lack. Editing and formatting what you have written is something I can do pretty quickly.

Somewhere on my to-do list is properly creating a contributing guide, I'll try to prioritize that to help you and others add notes.

**Comment by Cellivar** [comment]
Oh sidenote, this includes copy/pasting your previously written forum threads into documents here. I haven't done this yet since you expressed interest in contributing directly. Once I'm done with the notes I have about modules and other things I was going to start pulling in more information from things like the TCU thread.

**Comment by cr08** [comment]
Sorry for the delayed reply. All sounds good though and I'll add in stuff where I can. My attention span on this stuff lately is a bit erratic though so I'll probably leave the bulk of the work up to you and I'll fill stuff where I can.

**Comment by SeaWolfFan** [comment]
[FordPass Connect TCU Authorization Thru FORScan TCU-H.pdf](https://github.com/user-attachments/files/18548493/FordPass.Connect.TCU.Authorization.Thru.FORScan.TCU-H.pdf)
I came across this document while looking for instructions to upgrade the TCU to 4g, I think it would be a good addition to the projects folder.

**Comment by cr08** [comment]
> [FordPass Connect TCU Authorization Thru FORScan TCU-H.pdf](https://github.com/user-attachments/files/18548493/FordPass.Connect.TCU.Authorization.Thru.FORScan.TCU-H.pdf) I came across this document while looking for instructions to upgrade the TCU to 4g, I think it would be a good addition to the projects folder.

This is a great summary. It'll need to be tweaked slightly for the C-Max specifically around the AsBuilt numerical codes being listed as they refer to a gas only vehicle in that document. But nothing too major. Definitely worth adding IMO.

Apologies for not putting much more work forward on my end here. IRL stuff has gotten in the way the past couple years. I've since been really getting back into some of the C-Max communities and have some good ideas in mind for content to add so I'm gonna try to jump back into that work.
